core/src/plan/expr/function.rs

```diff
@@ -114,6 +115,7 @@ impl Function {
                 start: expr2,
                 count: Some(expr3),
             } => Exprs::Tripple([expr, expr2, expr3].into_iter()),
+            Self::Concat(exprs) => Exprs::VariableArgs(exprs.iter()),
```

Comment:
Nice decision to add `VariableArgs` to here :)
Let's also have a unit test for this.
You can add it to the below of this file.